Server.Tests/Characters/KitCarlsonSpecification.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Bang.Characters;
+using Bang.GameEvents;
+using Bang.Players;
+using FluentAssertions;
+using Server.Messages;
+using System;
+using Xunit;
+
+namespace Server.Tests.Characters
+{
+    public class KitCarlsonSpecification
+    {
+        [Fact]
+        public void Kit_Karlson_receives_choose_cards_response()
+        {
+            var game = CreateAndStartGame();
+            var nextPlayer = game.Gameplay.GetNextPlayer();
+            nextPlayer.SetInfo(game.Gameplay, nextPlayer.Role, new KitCarlson());
+
+            var nextPlayerTurnMessage = new NextPlayerTurnMessage(game.Gameplay.PlayerTurn);
+
+            var responseMessage = game.ProcessEvent(nextPlayerTurnMessage);
+            responseMessage.Should().BeOfType<ReplyActionMessage>();
+            var replayActionMessage = (ReplyActionMessage)responseMessage;
+            replayActionMessage.Response.Should().BeOfType<ChooseCardsResponse>();
+        }
+
+        [Fact]
+        public void Kit_Karlson_receives_3_cards_to_choose()
+        {
+            var game = CreateAndStartGame();
+            var nextPlayer = game.Gameplay.GetNextPlayer();
+            nextPlayer.SetInfo(game.Gameplay, nextPlayer.Role, new KitCarlson());
+
+            var nextPlayerTurnMessage = new NextPlayerTurnMessage(game.Gameplay.PlayerTurn);
+
+            var responseMessage = game.ProcessEvent(nextPlayerTurnMessage);
+            var replayActionMessage = (ReplyActionMessage)responseMessage;
+            var response = (ChooseCardsResponse)replayActionMessage.Response;
+            response.CardsToChoose.Count.Should().Be(3);
```

Comment:
Just for information: you can write `response.CardsToChoose.Should().HaveCount(3)` instead

Reply:
thanks, done